examples/advanced/edge/README.md

```diff
@@ -67,34 +82,83 @@ You will then see the device start receiving the model from the server and compl
 The server will perform aggregation and proceed to the next round.
 After the configured rounds have finished, the training is complete!
 
-## [Optional] End-to-end Cifar10 Example
+## [Optional] Local Proof-Of-Concept with DeviceSimulator: an End-to-end Cifar10 Example 
+Above we show how to run the mobile example with the NVFlare system on an actual device. For prototyping and testing a cross-device FL pipeline, 
+we usually do not start with real devices. Therefore, NVFlare provides DeviceSimulator to give simulated devices for testing the FL process
 
 Let's run an end-to-end example with Cifar10 dataset with baseline comparisons.
+
 ### Baselines
+First, let's run the centralized baseline on the whole dataset, and a 16-client federated baseline via NVFlare's standard single-layer pipeline using [JobAPI](https://nvflare.readthedocs.io/en/main/programming_guide/fed_job_api.html).
 1. Run the centralized baseline
 ```commandline
 cd baselines
 python cifar_train_central.py
+cd ..
 ```
 2. Run the federated baseline under regular single-layer setting
 ```commandline
+cd baselines
 python cifar_fl_base_job.py
+cd ..
+```
+Let's visualize the results:
+```commandline
+tensorboard --logdir=/tmp/nvflare/workspaces
 ```
+With the centralized training of 10 epochs, and the federated training of 10 rounds (1 local epoch per round), you should see the following results:
+<img src="./figs/cifar10.png" alt="Cifar10 Results" width="800" >
+The two learning will converge to similar accuracy, note that in this case each client holds partial data that is 1/16 of the whole training set sequentially split.
+
+### Simulated Cross-Device Federated Learning
```

Comment:
This section is duplicated of what's above.
if we want, maybe merge them, and explain they can do DeviceSimulator if they don't have a real iOS device?